Endace/Endace.manifest.json

```diff
@@ -0,0 +1,24 @@
+{
+    "author": "Endace",
+    "categories": [
+        "Compliance and Security",
+        "Sumo Logic Certified"
+    ],
+    "communityURL": "",
+    "description": "Bring clarity to every incident, alert, or issue with Endace packet capture evidence integrated directly into the Sumo Logic platform. Endace delivers scalable, always-on packet capture in on-premise and hybrid cloud environments for definitive network visibility.  Packets are the ultimate tamperproof source of truth that accelerates incident response for security threats, outages, and performance issues. This integration provides your operations team with fast access to the network packets (pcap) related to any incident or threat for rapid and precise incident response. ",
+    "helpURL": "https://github.com/Endace/Integrations.SumoLogic/blob/main/README.md",
```

Comment:
this helpurl is throwing 404 for me will it be live after we publish the app?

Reply:
@asimonsendace - Can you confirm this is working?